From 4fe04785b9afb39ec17ff35807e9c2c2b16917ae Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Fri, 15 Apr 2016 14:02:47 +0000
Subject: [PATCH 0743/1077] MANTIS 0040920 extension: additional error codes
 ManualMode; SwitchedOff

codes.idl
V1.4.1
- added materna extension BAGGAGE_MANUAL_MODE, BAGGAGE_SWITCHED_OFF

ConveyorDev.idl
V1.11
- added ERR_MANUAL_MODE, ERR_SWITCHED_OFF


git-svn-id: svn://localhost/SelfServiceCommon/trunk@1006 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Cuss/codes.idl         | 8 ++++++++
 SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl | 6 ++++--
 2 files changed, 12 insertions(+), 2 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Cuss/codes.idl b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
index c4b77bd3..1384a2b5 100644
--- a/SelfServiceCommon/Massai/idl/Cuss/codes.idl
+++ b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
@@ -68,6 +68,11 @@ Changed:   Added baggage handling status codes (Nicolas Bruns, 29.06.2004)
   - added DS_TYPES_KEY, DS_TYPES_KEY_UP, DS_TYPES_KEY_DOWN
   - fixed DS_TYPES_SSML_11
   - added materna extension BAGGAGE_INTRUSION, BAGGAGE_INTRUSION_FRONT, BAGGAGE_INTRUSION_BACK, BAGGAGE_TRANSPORT_BUSY, BAGGAGE_MISTRACKED, BAGGAGE_UNEXPECTED_CHANGE, BAGGAGE_ACCEPTED, BAGGAGE_DELIVERED, BAGGAGE_INTERFERENCE_USER, BAGGAGE_INTRUSION_SAFETY, BAGGAGE_INCORRECTLY_PLACED, BAGGAGE_TUB_REQUIRED, BAGGAGE_HANDLE_EXTENDED, BAGGAGE_TOO_WIDE, BAGGAGE_TOO_SMALL,               
+@date 15.04.2016
+@li V1.4.1
+@li Mantis 0040920: Conveyor3Dev: additional error codes ManualMode; SwitchedOff
+- added materna extension BAGGAGE_MANUAL_MODE, BAGGAGE_SWITCHED_OFF
+
 */
 
 #ifndef CODES_IDL
@@ -343,6 +348,9 @@ module statuscodes
     const long BAGGAGE_HANDLE_EXTENDED       = 172; /**< <b>Scope:</b> private + platform; materna ccm extension */
     const long BAGGAGE_TOO_WIDE              = 173; /**< <b>Scope:</b> private + platform; materna ccm extension */
     const long BAGGAGE_TOO_SMALL             = 174; /**< <b>Scope:</b> private + platform; materna ccm extension */
+    const long BAGGAGE_MANUAL_MODE           = 175; /**< <b>Scope:</b> private + platform; materna ccm extension */
+    const long BAGGAGE_SWITCHED_OFF          = 176; /**< <b>Scope:</b> private + platform; materna ccm extension */
+
 
     const long FORMAT_ERROR                = 201; /**< <b>Scope:</b> private + platform (platform only for output classes) */
     const long LENGTH_ERROR                = 202; /**< <b>Scope:</b> private + platform (platform only for output classes) */
diff --git a/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl b/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
index 3b7e8b62..fcec24da 100644
--- a/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
@@ -39,7 +39,7 @@
 **/
 module conveyordev
 {
-   const string interface_version = "1.10";
+   const string interface_version = "1.11";
 /*
 using DeviceComponents::indicationListener and DeviceComponents::Generic from mdci.idl
   RC indicationListener::indication(in any ind);
@@ -183,7 +183,9 @@ using DeviceComponents::indicationListener and DeviceComponents::Generic from md
       ERR_INTRUSION,                /**< Safety intrusions of unknown kind. */
       ERR_FRONT_INTRUSION,          /**< Front-side safety intrusion. */
       ERR_BACK_INTRUSION,           /**< Back-side safety intrusion. */
-      ERR_PIECE_MISTRACKED          /**< Baggage tracking failure. */
+      ERR_PIECE_MISTRACKED,         /**< Baggage tracking failure. */
+      ERR_MANUAL_MODE,              /**< Online, manual mode */
+      ERR_SWITCHED_OFF              /**< Online, switched off */
    };
 
    /**
-- 
2.41.0.windows.1

